samples/documentation/asciidoc/index.adoc

```diff
@@ -0,0 +1,1659 @@
+= OpenAPI Petstore 
+team@openapitools.org
+1.0.0
+:toc: left
+:numbered:
+:toclevels: 3
+:source-highlighter: highlightjs
+:keywords: openapi, rest, OpenAPI Petstore 
+:specDir: modules\openapi-generator\src\main\resources\asciidoc-documentation
+:snippetDir: .
+
+[abstract]
+.Abstract
+This is a sample server Petstore server. For this sample, you can use the api key `special-key` to test the authorization filters.
+
+
+include::stubs/empty.adoc[]
+
+
+== Endpoints
+
+
+[.Pet]
+=== Pet
+
+
+[.addPet]

```

Comment:
@man-at-home we may need to prefix it with the class name (e.g. Pet_addPet) as multiple classes can have the same method (e.g. both Pet and User have the same method `getId`: Pet_getId, User_getId).

Reply:
I will look into it. At the moment the generated doc does not use these tags, but the idee would be to allow links to specific calls (if the generated docs format is linkable like html)